### DISCUSSION TITLE
Is it possible to update an another atom from a selector?

### DISCUSSION THREAD

**Discussion by rafaeltuelho** [facebookexperimental/Recoil#1484] [Q&A] [answered]
I want to update another atom from inside a selector/atom set function. Given AtomA and SelectorB whenever I update SelectorB I want to update AtomA.
Is it possible? If yes is it considered an anti-pattern?

**Comment by drarmstr** [comment, marked as answer]
Yes, it is possible to update multiple upstream atoms or selectors from a writeable selector's `set` callback.
